README.md

```diff
@@ -10,12 +10,18 @@ The recommended and official way to run `image-build` is using the `ghcr.io/open
 To build an image using the container, the config file needs to be mapped into the container, as well as the FUSE filesystem device:
 
 ```
-podman run \
-  --rm \
+podman run --rm \
   --device /dev/fuse \
-  -v /path/to/config.yaml:/home/builder/config.yaml \
+  --network host \
+  --cap-add=SYS_ADMIN \
+  --cap-add=SETUID \
+  --cap-add=SETGID \
+  --security-opt seccomp=unconfined \
+  --security-opt label=disable \
+  --userns=keep-id \
```

Comment:
```suggestion
  --userns=keep-id:uid=1000,gid=1000 \
```
I've been using this to ensure my outside UID is mapped to `builder` so that `builder` can access things like 0600 files.